lib/index.js

```diff
@@ -882,7 +882,7 @@ exports.isInteger = function (value) {
 
     return (typeof value === 'number' &&
             parseFloat(value) === parseInt(value, 10) &&
```

Comment:
do we actually need anything other than `Number.isSafeInteger`? it returns false for any input that isn't a number too, so i'm pretty sure `exports.isInteger = Number.isSafeInteger` would be just as effective